velox/type/Timestamp.cpp

```diff
@@ -44,6 +45,12 @@ inline int64_t getPrestoTZOffsetInSeconds(int16_t tzID) {
 } // namespace
 
 void Timestamp::toTimezone(const date::time_zone& zone) {
+  // Magic number -2^39 + 24*3600. Passing this number time_zone::to_sys() will
+  // SIGABRT. We don't want that to happen.
```

Comment:
1. How do we know about it?
2. Why does time_zone::to_sys() do it?
3. Suggestion: "Passing this number time_zone::to_sys() will" -> "This number and any number lower than that  will cause time_zone::to_sys() to". As I initially thought that it is only that number, but the code says otherwise.

Reply:
1. I got to know about it when I wrote unit test and got that number from it.

Reply:
The comment has not been corrected.
Btw, does documentation on time_zone::to_sys() or internet say anything about it?